keymanager-api-common/src/api.rs

```diff
@@ -305,6 +281,28 @@ impl Default for TrustedPolicySigners {
     }
 }
 
+impl TrustedPolicySigners {
+    /// Verify that policy has valid signatures and that enough of them are from trusted signers.
+    pub fn verify_and_trust(&self, policy: &SignedPolicySGX) -> Result<(), KeyManagerError> {
+        policy
+            .verify()
+            .map_err(|err| KeyManagerError::PolicyInvalid(err.into()))?;
+
+        self.trust(policy)
+    }
+
+    /// Verify that policy has enough signatures from trusted signers.
+    fn trust(&self, policy: &SignedPolicySGX) -> Result<(), KeyManagerError> {
```

Comment:
This makes sense.

Maybe just rename to something else as `trust` makes it sound like this will make the policy trusted instead of verifying the threshold.

Reply:
Renamed to `verify_trusted_signers`.